<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="Ramakant" failures="1" tests="3" name="Test(failed)" time="0.053" errors="0" timestamp="2024-06-06T02:03:22 IST">
  <testcase classname="seleniumTesting.TestNG4" name="a" time="0.013">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at seleniumTesting.TestNG4.a(TestNG4.java:9)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- a -->
  <testcase classname="seleniumTesting.TestNG4" name="b" time="0.0">
    <skipped/>
  </testcase> <!-- b -->
  <testcase classname="seleniumTesting.TestNG3$TestNG2" name="b1" time="0.0">
    <skipped/>
  </testcase> <!-- b1 -->
  <testcase classname="seleniumTesting.TestNG3$TestNG2" name="testBefore" time="0.0">
    <ignored/>
  </testcase> <!-- testBefore -->
  <testcase classname="seleniumTesting.TestNG3$TestNG2" name="test" time="0.0">
    <ignored/>
  </testcase> <!-- test -->
  <testcase classname="seleniumTesting.TestNG3$TestNG2" name="test4" time="0.0">
    <ignored/>
  </testcase> <!-- test4 -->
  <testcase classname="seleniumTesting.TestNG3$TestNG2" name="testAfter" time="0.0">
    <ignored/>
  </testcase> <!-- testAfter -->
</testsuite> <!-- Test(failed) -->
